OpenRA.Mods.Cnc/UtilityCommands/ListMegContentsCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.Mods.Cnc.FileFormats;
+using OpenRA.Mods.Cnc.FileSystem;
+
+namespace OpenRA.Mods.Cnc.UtilityCommands
+{
+	class ListMegContentsCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--list-meg"; } }
```

Comment:
We can't extract from the meg files from the content installer, so we won't need this command.